diff --git a/dist/pptxgen.cjs.js b/dist/pptxgen.cjs.js
index 143dfa3e7fb54054153bad16b6c991697c0d31e7..f43edce9abee257124006b59c480771075ac7e21 100644
--- a/dist/pptxgen.cjs.js
+++ b/dist/pptxgen.cjs.js
@@ -6696,16 +6696,14 @@ function makeXmlPresentation(pres) {
         `xmlns:p="http://schemas.openxmlformats.org/presentationml/2006/main" ${pres.rtlMode ? 'rtl="1"' : ''} saveSubsetFonts="1" autoCompressPictures="0">`;
     // STEP 1: Add slide master (SPEC: tag 1 under <presentation>)
     strXml += '<p:sldMasterIdLst><p:sldMasterId id="2147483648" r:id="rId1"/></p:sldMasterIdLst>';
-    // STEP 2: Add all Slides (SPEC: tag 3 under <presentation>)
-    strXml += '<p:sldIdLst>';
-    pres.slides.forEach(slide => (strXml += `<p:sldId id="${slide._slideId}" r:id="rId${slide._rId}"/>`));
-    strXml += '</p:sldIdLst>';
-    // STEP 3: Add Notes Master (SPEC: tag 2 under <presentation>)
+    // STEP 2: Add Notes Master (SPEC: tag 2 under <presentation>)
     // (NOTE: length+2 is from `presentation.xml.rels` func (since we have to match this rId, we just use same logic))
-    // IMPORTANT: In this order (matches PPT2019) PPT will give corruption message on open!
-    // IMPORTANT: Placing this before `<p:sldIdLst>` causes warning in modern powerpoint!
     // IMPORTANT: Presentations open without warning Without this line, however, the pres isnt preview in Finder anymore or viewable in iOS!
     strXml += `<p:notesMasterIdLst><p:notesMasterId r:id="rId${pres.slides.length + 2}"/></p:notesMasterIdLst>`;
+    // STEP 3: Add all Slides (SPEC: tag 3 under <presentation>)
+    strXml += '<p:sldIdLst>';
+    pres.slides.forEach(slide => (strXml += `<p:sldId id="${slide._slideId}" r:id="rId${slide._rId}"/>`));
+    strXml += '</p:sldIdLst>';
     // STEP 4: Add sizes
     strXml += `<p:sldSz cx="${pres.presLayout.width}" cy="${pres.presLayout.height}"/>`;
     strXml += `<p:notesSz cx="${pres.presLayout.height}" cy="${pres.presLayout.width}"/>`;
diff --git a/dist/pptxgen.es.js b/dist/pptxgen.es.js
index 449038cbf3d367c7e5a51bf692793a25bd06a86e..6491130671bcfae1ff3b2c1116ce0e55cad8eb6b 100644
--- a/dist/pptxgen.es.js
+++ b/dist/pptxgen.es.js
@@ -6694,16 +6694,14 @@ function makeXmlPresentation(pres) {
         `xmlns:p="http://schemas.openxmlformats.org/presentationml/2006/main" ${pres.rtlMode ? 'rtl="1"' : ''} saveSubsetFonts="1" autoCompressPictures="0">`;
     // STEP 1: Add slide master (SPEC: tag 1 under <presentation>)
     strXml += '<p:sldMasterIdLst><p:sldMasterId id="2147483648" r:id="rId1"/></p:sldMasterIdLst>';
-    // STEP 2: Add all Slides (SPEC: tag 3 under <presentation>)
-    strXml += '<p:sldIdLst>';
-    pres.slides.forEach(slide => (strXml += `<p:sldId id="${slide._slideId}" r:id="rId${slide._rId}"/>`));
-    strXml += '</p:sldIdLst>';
-    // STEP 3: Add Notes Master (SPEC: tag 2 under <presentation>)
+    // STEP 2: Add Notes Master (SPEC: tag 2 under <presentation>)
     // (NOTE: length+2 is from `presentation.xml.rels` func (since we have to match this rId, we just use same logic))
-    // IMPORTANT: In this order (matches PPT2019) PPT will give corruption message on open!
-    // IMPORTANT: Placing this before `<p:sldIdLst>` causes warning in modern powerpoint!
     // IMPORTANT: Presentations open without warning Without this line, however, the pres isnt preview in Finder anymore or viewable in iOS!
     strXml += `<p:notesMasterIdLst><p:notesMasterId r:id="rId${pres.slides.length + 2}"/></p:notesMasterIdLst>`;
+    // STEP 3: Add all Slides (SPEC: tag 3 under <presentation>)
+    strXml += '<p:sldIdLst>';
+    pres.slides.forEach(slide => (strXml += `<p:sldId id="${slide._slideId}" r:id="rId${slide._rId}"/>`));
+    strXml += '</p:sldIdLst>';
     // STEP 4: Add sizes
     strXml += `<p:sldSz cx="${pres.presLayout.width}" cy="${pres.presLayout.height}"/>`;
     strXml += `<p:notesSz cx="${pres.presLayout.height}" cy="${pres.presLayout.width}"/>`;
